NewSpring/ApollosProject
# (DEV-3928) Fixes Prayer List Warning

## DESCRIPTION

<img width="447" alt="Screen Shot 2019-07-22 at 11 06 36 AM" src="https://user-images.githubusercontent.com/2659478/61643538-18de1400-ac71-11e9-82f7-0ee1b4db9537.png">


### What does this PR do, or why is it needed?

Fixes an expected prop type into the `SaveButton`

### How do I test this PR?

Just open any prayer list and make sure there's no warning.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
